README.md

```diff
@@ -33,4 +37,4 @@ This is an implementation of the `largeVis` algorithm described in (<https://arx
 
 #### Building Notes
 
--   **Note on R 3.4**: Before R 3.4, the CRAN binaries were likely to have been compiled without OpenMP, and getting OpenMP to work on Mac OS X was somewhat tricky. This should all have changed (for the better) with R 3.4, which natively using `clang 4.0` by default. Since R 3.4 is new, I'm not able to provide advice, but am interested in hearing of any issues and any workarounds to issues that you may discover.
+-   **Note on R 3.4**: Before R 3.4, the CRAN binaries were likely to have been compiled without OpenMP, and getting OpenMP to work on Mac OS X was somewhat tricky. This should all have changed (for the better) with R 3.4, which natively using `clang 4.0` by default. 
```

Comment:
I believe there's another change relevant to this in 4.0, because the toolchain used to make packages for OS X changed again.  (Is it possible for you to take a peek?)

Also, the vignettes are automatically regenerated as part of the submission process.  I haven't been including them in each git update because there isn't a point.

Reply:
> I believe there's another change relevant to this in 4.0, because the toolchain used to make packages for OS X changed again. 

That's my understanding as well---I'll need to investigate this. 

> Also, the vignettes are automatically regenerated as part of the submission process. I haven't been including them in each git update because there isn't a point.

I often agree with this, but I've lately found rendered files are useful for github; it's now each for users to view these markdown files and follow along via the browser. Not rendering the vignettes is fine too.